src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -1,36 +1,58 @@
 <template>
-    <modal :open="open">
-        <h4 v-translate="'haveYouSaved'"></h4>
-        <p v-translate="'discardWarning'"></p>
-        <ul>
-            <li v-for="step in laterCompleteSteps">
-                Step {{step.number}}: <span v-translate="step.textKey"></span>
-            </li>
-        </ul>
-        <p v-translate="'savePrompt'"></p>
-        <template v-slot:footer>
-            <button type="button"
-                    class="btn btn-white"
-                    @click="continueEditing"
-                    v-translate="'discardSteps'">
-            </button>
-            <button type="button"
-                    class="btn btn-red"
-                    @click="cancelEditing"
-                    v-translate="'cancelEdit'">
-            </button>
-        </template>
-    </modal>
+    <div>
+        <modal :open="open">
+            <h4 v-if="guestUser" v-translate="'haveYouSaved'"></h4>
+            <h4 v-if="!guestUser">Save snapshot?</h4>
+
+            <p v-translate="'discardWarning'"></p>
+            <ul>
+                <li v-for="step in laterCompleteSteps">
+                    Step {{step.number}}: <span v-translate="step.textKey"></span>
+                </li>
+            </ul>
+
+            <p v-if="guestUser"  v-translate="'savePrompt'"></p>
+            <p v-if="!guestUser">
+                These steps will automatically be saved in a snapshot. You will be able to reload this snapshot from the Versions page.
```

Comment:
The logic all seems fine to me here, re the wording I think we discussed ditching "snapshot" as a user facing term?

Reply:
Pending with Jeff atm. Ticket for changing user-facing terms as decided: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1745